packages/react/src/patterns/sub-patterns/CardLink/README.stories.mdx

```diff
@@ -0,0 +1,90 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { CardLink } from './CardLink';
+
+# CardLink
+
+> The cardlink component uses
+> [Carbon's Clickable Tile component](https://www.carbondesignsystem.com/components/tile/code#clickable-tile)
+> while presenting content in a concise and readable style.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### CSS
+
+```css
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/card/index.scss';
+```
+
+> 💡 Only import fonts once per usage. Don't forget to import the Card styles
+> from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+##### JS
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { CardLink } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <CardLink
+      card={{
+        heading: 'Heading',
+        copy: 'Lorem ipsum dolor sit amet',
+        cta: {
+          href: 'http://www.example.com',
+          icon: {
+            src: ArrowRight20,
+          },
+        },
+      }}
+    />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+Add the following line in your `.env` file at the root of your project.
+[See more details](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/styles#usage).
+
+```
+  SASS_PATH=node_modules:src
+```
+
+> 💡 Don't forget to import the card styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+<Props of={(console.log('CardLink:', CardLink), CardLink)} />
+
+### cta (required)
+
+| Name   | Data Type | Description                                                                                                                      |
+| ------ | --------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `href` | String    | Valid URL for a the location of an internal or external resource                                                                 |
+| `icon` | String    | Provide an optional icon to the footer from [Carbon's icon library](https://www.carbondesignsystem.com/guidelines/icons/library) |
+| `copy` | String    | Optional text for CTA                                                                                                            |
```

Comment:
Can we remove this if we are referencing the prop table?

Reply:
Oh, yep we can